test/e2e/dra/dra.go

```diff
@@ -2646,6 +2736,29 @@ func (b *builder) setUp(ctx context.Context) {
 	b.podCounter = 0
 	b.claimCounter = 0
 	b.create(ctx, b.class())
+	// DRAExtendedResource requires scheduler to have 'create' permission on resourceclaim.
+	role, err := b.f.ClientSet.RbacV1().ClusterRoles().Get(ctx, "system:kube-scheduler", metav1.GetOptions{})
```

Comment:
This doesn't belong into the test. Instead, extend `plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go` with conditional additional permissions depending on the feature gate.

Reply:
done